camelot/view/controls/delegates/customdelegate.py

```diff
@@ -214,6 +236,11 @@ def setEditorData(self, editor, index):
         editor.set_field_attributes(**field_attributes)
         editor.set_value(value)
 
+        # update actions
+        post(self.update_actions_request,
+             self.update_actions_response,
+             args=(editor, index))
```

Comment:
Nu worden de actions altijd geupdate, voorheen enkel als set_field_attributes of set_value explicitet update_actions() aanriepen.

Reply:
Eventueel kan ik wel die index meegeven aan set_field_attributes en set_value om dan enkel de actions te updaten indien nodig...